### ISSUE TITLE
Naming issue?

### ISSUE THREAD

**Issue by delta1** [Nubotics/generator-arc-static#8]
Just so you're aware of the Arc programming language

http://en.wikipedia.org/wiki/Arc_%28programming_language%29 

It hasn't really seen widespread adoption, but is relatively well known because of Paul Graham and the Hacker News scene. 

I'm not too bothered since it's called Arc Static, right? But there is potential confusion, any thoughts?

**Comment by Nubuck** [comment]
Yeah man, I am totally aware - arc at this point is just an R&D working title - for lack of a better name.

I think this is a good place to discuss a name for the project collection - arc essentially came from the idea of the arc of electricity between 2 conductors. 

Our goal with the arc projects is a collection of next generation modular, composable, pluggable and extendable libraries, platforms and generators - all connecting through a common interface.

Open to suggestions for an official title....

**Comment by MartinMcCabe** [comment]
Hey Guys

Slightly off topic, but I'm curious to find out...
What versions of Node and NPM are you all running? I'm building a site
using the generator and it was working 100% on my first machine. I then got
a "new" laptop and so I got busy installing all the shit I need, starting
with Node. But then certain modules wouldn't install properly, like
Browser-sync and some of its dependencies.
Also, when compiling the CSS (running gulp watch) some of my classes
stopped working. When I inspect the CSS in developer tools there's some
\u200b characters in red.

Eventually after trying a bunch of things I uninstalled Node 0.12 (with NPM
2.5.1) and rolled back to Node 0.10.36 with NPM 1.4.28 and everything work
fine now.

**Comment by WillemLabu** [comment]
```
$ node -v
v0.10.33

$ npm -v
1.4.28
```
So, I sketched out a few notes on a name.. Let me know what you guys think.

This tool will become the starting point for projects, as well as the tool we use to create new sections/pods/modules/routes for existing projects.

This is the meta that generates the real. Speaking biologically, this could be seen as DNA.
Tying arc back into things.. I read a really interesting article about how [electricity plays a part in biological (re)generation](https://medium.com/matter/could-this-man-hold-the-secret-to-human-regeneration-1e66944f0a8d) recently.

Thinking about this and how electricity's used in biology lead me to thing about synapses and then cellular biology.

Cell biology used to be known as _cytology_ from the Greek "kytos" meaning "contain".

All that said, I like 'arc' because it makes me think of this:

![arc reactor](http://img3.wikia.nocookie.net/__cb20121215194619/marveldatabase/images/4/45/Arc_Reactor_(Earth-199999)_miniaturized_third_version.jpg)

**Comment by Nubuck** [comment]
Awesome Willem!

If im honest the name actually came from Tony Stark's Arc reactor - because my ambition with the arc project collection is to create a machine intelligent CLI tool - called Jarvis... or Javas lol.

Everybody thinks Stark's greatest creation is his Iron Man suit - but it's actually Jarvis - without Jarvis he'd have a basement of Chinese kids building his suits.

**Comment by Nubuck** [comment]
if we meet resistance on the name i quite like the name "cyto"

**Comment by WillemLabu** [comment]
Well.. He created a new element.. But I'm not sure that's "inventing".

20 minutes and 10 wiki articles later.. lolkek

The element was called [Vibranium](http://marvelcinematicuniverse.wikia.com/wiki/Vibranium) - and Stark later tried to get it patented as "badassium" but ran into many legal issues.

**Comment by Nubuck** [comment]
Vibranium is a bit of a risque name haha

**Comment by delta1** [comment]
$ node -v
    v0.10.28
    
    $ npm -v
    2.3.0

I'm really not too bothered about the name, I like Arc and Cyto, even Kytos, or Stark? ha ha

Insane in the Vibranium... lol 

It's about as difficult as choosing a band name :) Just start making stuff up like Cytark

**Comment by WillemLabu** [comment]
I actually like arc the most, and there are bound to be [name-clashes](http://stackoverflow.com/questions/2337412/why-jquery-is-called-jquery-what-is-the-name-reason).
